File: tests/openmp_examples/data_environment/udr.1.c
#pragma omp declare reduction(min : struct point) combiner( minproc(&omp_out, &omp_in) ) initializer( omp_priv = { 2147483647, 2147483647 } )
#pragma omp declare reduction(max : struct point) combiner( maxproc(&omp_out, &omp_in) ) initializer( omp_priv = { 0, 0 } )
#pragma omp parallel for reduction(min:minp) reduction(max:maxp)

File: tests/openmp_examples/data_environment/udr.1.f90
!$omp   declare reduction(min : point) combiner(minproc(omp_out, omp_in)) initializer(omp_priv = point(HUGE(0), HUGE(0)))
!$omp   declare reduction(max : point) combiner(maxproc(omp_out, omp_in)) initializer(omp_priv = point(0, 0))
!$omp   parallel do reduction(min:minp) reduction(max:maxp)
